test.py

```diff
@@ -1,10 +1,26 @@
+import os
 import time
 
+from dotenv import load_dotenv
+
 from proxyllm import LLMProxy
+from proxyllm.provider.anthropic import claude
 
 # NOTE: Temp Test file, will be removed in future in favour of unit/integration tests
 
 
+def request() -> None:
```

Comment:
Testing like this shouldn't be pushed, especially since test.py is the file that we use for a quick integration test. It's best to remove code and print statements before pushing.

Reply:
Ok, I have made that removal